docs/v3.x/deployment/ibm-cloud.md

```diff
@@ -0,0 +1,332 @@
+# IBM Cloud
+
+This is a step-by-step guide for deploying a Strapi project on [IBM Cloud](https://www.ibm.com/cloud). Among the available alternatives, this guide covers the deployment on [IBM Cloud Foundry](https://www.cloudfoundry.org/the-foundry/ibm-cloud-foundry) using the official databases services.
+
+### IBM Cloud Install Requirements
+
+- You must have a [free IBM Cloud account](https://cloud.ibm.com/registration) before doing these steps.
+
+If you already have the IBM Cloud CLI installed locally on your computer. Skip to [Login to IBM Cloud](#_2-login-to-ibm-cloud-from-your-cli).
+
+#### 1. IBM Cloud CLI Installation
+
+Download and install the [CLI](https://cloud.ibm.com/docs/cli) for your operating system:
+
+:::: tabs
+
+::: tab "Mac O/S"
+Run the following from your terminal:
+
+```bash
+curl -sL https://ibm.biz/idt-installer | bash
+```
+
+:::
+
+::: tab Ubuntu
+Run the following from your terminal:
+
+```bash
+curl -sL https://ibm.biz/idt-installer | bash
+```
+
+:::
+
+::: tab Windows
+Run the following in Powershell as administrator:
+
+```bash
+[Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls11, Tls, Ssl3"; iex(New-Object Net.WebClient).DownloadString('https://ibm.biz/idt-win-installer')
```

Comment:
May want to provide an alternative using an executable (if it's an option, not sure) as not everyone may be entirely familiar with powershell

Reply:
External documentation might be good here too.

Reply:
I changed the link to point to the installation instructions. I think it should be enough.

An executable is not provided.